RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2151,10 +2155,22 @@ boolean L11_hiddenCityZones()
 		}
 		return false;
 	}
+	boolean equipTearawayPants()
+	{
+		if(in_ag() || is_boris() || in_wotsf())
```

Comment:
I'd prefer not to list the paths which don't get a machete here as well. Possible to forget in future updates

Reply:
I did boris and wotsf because those still have combat skills unlike pokefam but can't use the machete. I'll this below the canUseMachete boolean and exclude pokefam from this instead, so it's exclusionary of paths that can't use skills but inclusionary of paths that can't use/don't want to use machetes